app/role_data.go

```diff
@@ -287,6 +291,10 @@ func addRoles(management *config.ManagementContext) (string, error) {
 	rb.addRoleTemplate("View Project Catalogs", "projectcatalogs-view", "project", true, false, false, false).
 		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("get", "list", "watch")
 
+	rb.addRoleTemplate("Project Monitoring View", "project-monitoring-view", "project", true, false, false, false).
```

Comment:
why do we need this role

Reply:
i now see that this is the role you are giving the serviceaccount

Reply:
its a little weird that we are adding a roleTemplate when we dont intend for it to ever be used by a Project or Cluster RoleTemplateBinding.